File: DokkyFlights.API/ReadModels/BookingRm.cs
namespace DokkyFlights.API.ReadModels
{
    public record BookingRm(
        Guid FlightId,
        string Airline,
        string Price,
        TimePlaceRm Arrival,
        TimePlaceRm Departure,
        int NumberOfSeats,
        string PassengerEmail);
}

File: DokkyFlights/src/app/api/models/booking-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from '../models/time-place-rm';
export interface BookingRm {
  airline?: null | string;
  arrival?: TimePlaceRm;
  departure?: TimePlaceRm;
  flightId?: string;
  numberOfSeats?: number;
  passengerEmail?: null | string;
  price?: null | string;
}
